lib/model.js

```diff
@@ -2101,9 +2104,10 @@ class Model {
         // Map attributes for serial identification
         const attributes = {};
         for (const attr in this.tableAttributes) {
-          attributes[attr] = this.rawAttributes[attr];
           if (this.rawAttributes[attr].field) {
             attributes[this.rawAttributes[attr].field] = this.rawAttributes[attr];
```

Comment:
As `field` is always set , we can do `attributes[this.rawAttributes[attr].field]` ?